src/components/Send.jsx

```diff
@@ -327,6 +343,133 @@ export default function Send({ makerRunning, coinjoinInProcess }) {
     }
   }
 
+  const balanceBreakdown = (accountNumber) => {
+    if (!walletInfo || !walletInfo.accounts) {
+      return null
+    }
+
+    const filtered = walletInfo.accounts.filter((account) => {
+      return parseInt(account.account, 10) === accountNumber
+    })
+
+    if (filtered.length !== 1) {
+      return null
+    }
+
+    const utxosByAccount = (utxos || []).reduce((acc, utxo) => {
+      acc[utxo.mixdepth] = acc[utxo.mixdepth] || []
+      acc[utxo.mixdepth].push(utxo)
+      return acc
+    }, {})
+    const accountUtxos = utxosByAccount[accountNumber] || []
+    const frozenOrLockedUtxos = accountUtxos.filter((utxo) => utxo.frozen || utxo.locktime)
+    const balanceFrozenOrLocked = frozenOrLockedUtxos.reduce((acc, utxo) => acc + utxo.value, 0)
+
+    return {
+      totalBalance: btcToSats(filtered[0].account_balance),
+      frozenOrLockedBalance: balanceFrozenOrLocked,
+    }
+  }
+
+  const amountFieldValue = () => {
+    if (amount === null || Number.isNaN(amount)) return ''
+
+    if (isSweep) {
+      const breakdown = balanceBreakdown(account)
+
+      if (!breakdown) return ''
+
+      return breakdown.totalBalance - breakdown.frozenOrLockedBalance
+    }
+
+    return amount
+  }
+
+  const frozenOrLockedWarning = () => {
+    const breakdown = balanceBreakdown(account)
+
+    if (!breakdown) return null
+
+    return (
+      <div className="sweep-breakdown mt-1">
+        <rb.Accordion flush>
+          <rb.Accordion.Item eventKey="0">
+            <rb.Accordion.Header>
+              <div className="d-flex align-items-center justify-content-end w-100">
+                {t('send.button_sweep_amount_breakdown')}
+              </div>
+            </rb.Accordion.Header>
+            <rb.Accordion.Body className="px-0">
+              <table class="table table-sm">
```

Comment:
```suggestion
              <table className="table table-sm">
```

Reply:
Also, the text inside `table` is invisible in dark mode. (`color: #212529;` in _tables.scss)

Reply:
Yep still have to test dark mode!